core/src/main/java/org/commonjava/indy/core/content/DefaultDownloadManager.java

```diff
@@ -97,6 +97,9 @@
     @Inject
     private StoreDataManager storeManager;
 
+    @Inject
+    private ContentAdvisor contentAdvisor;
```

Comment:
Currently issue is that there is a unit-test in http-proxy depends on constructor of this class, which also needs to initialize this field in that ctor. So if we use Instance<T> that means we will need a mechanism to generate a Mock Instance in that unit-testing. Seems that there isn't now at all. Do we need to introduce a Mock framework like Mockito?